app/Http/Controllers/PostsController.php

```diff
@@ -69,7 +69,9 @@ public function __construct(PostManager $posts, SignupManager $signups, PostTran
     public function index(Request $request)
     {
         $query = $this->newQuery(Post::class)
-            ->orderBy('created_at', 'desc');
+            ->orderBy('created_at', 'desc')
+            // Eagerly load the `signup` relationship.
+            ->with('signup');
```

Comment:
We only want to eager-load this if `?include=signup` right?

Reply:
ah yes! Thanks for catching that, @DFurnes ! I updated and deployed that to QA to test that it still worked and looks good! :)